mk

```diff
@@ -61,17 +55,6 @@ GIT_CLONE_DIR = $(HOME)/.ooniprobe-build/src
 $(GIT_CLONE_DIR):
 	mkdir -p $(GIT_CLONE_DIR)
 
-#help:
-#help: * GOLANG_DOCKER_GOCACHE : where to store golang's build cache to
-#help:                           speed up subsequent Docker builds.
-GOLANG_DOCKER_GOCACHE = $(HOME)/.ooniprobe-build/docker/gocache
-
-#help:
-#help: * GOLANG_DOCKER_GOPATH  : GOPATH directory used by builds running
-#help:                           inside docker to significantly speed
-#help:                           up subsequent Docker based builds.
-GOLANG_DOCKER_GOPATH := $(HOME)/.ooniprobe-build/docker/gopath
-
```

Comment:
Removing caching since it has no impact on cloud builds speed and "caching is hard"